provision/kubernetes/helpers.go

```diff
@@ -558,10 +559,12 @@ func runPod(args runSinglePodArgs) error {
 			RestartPolicy: apiv1.RestartPolicyNever,
 			Containers: []apiv1.Container{
 				{
-					Name:    args.name,
-					Image:   args.image,
-					Command: args.cmds,
-					Env:     args.envs,
+					Name:      args.name,
+					Image:     args.image,
+					Command:   []string{"sh", "-c", "cat >/dev/null && " + args.cmd},
```

Comment:
I think we can use `-ec` here, `imageTagAndPush` used to rely on `-e` to stop after a possible docker pull error.